ghash/Cargo.toml

```diff
@@ -19,7 +19,7 @@ opaque-debug = "0.3"
 polyval = { version = "0.5.1", path = "../polyval" }
 
 # optional dependencies
-zeroize = { version = ">=1, <1.4", optional = true, default-features = false }
+zeroize = { version = "1.5", optional = true, default-features = false }
```

Comment:
Since there aren't any 1.5-specific features being used, I'd suggest (in all cases) just changing this to `1`

Reply:
Sounds good. Just saw that this doesn't really work with the MSRV, should I just bump the MSRV too?

Reply:
Yeah that's fine. `cipher` v0.4 is MSRV 1.56 so bumping these as well makes sense

Reply:
Done, I upgraded the Rust edition while I was at it.